substrate/frame/revive/fixtures/contracts/call_self_with_dust.sol

```diff
@@ -0,0 +1,7 @@
+contract CallSelfWithDust {
+    function f() external payable {}
```

Comment:
To check, if `f` here contained some code, this code will be executed and the only thing that we have a `NOOP` for is just the transfers, is that correct?